.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
Do we really want to force Windows users to use LF?

On IDEs (e.g. VS Code) it's probably not a problem, but on any basic plain text editor it might become a nuisance. In theory, using the `.gitattributes` should be enough so that no file is committed with CRLF.

See the note on [the EditorConfig wiki](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#end_of_line):

> NOTE: if you want to use native line endings between different operating systems it is better not to set this option and leave that task to the VCS! In the future we might add a value like native for this scenario (cf [#226](https://github.com/editorconfig/editorconfig/issues/226)).

Reply:
Indeed, to preserve and force our repository to use LF only, .gitattributes is all what we need.
The intention here is to preserve Windows developers from having their linters complaining on every line for that extra CR.
This should only affect editors that integrate the .editorconfig file settings in their configuration, leaving the rest to perform natively.